solver.rb

```diff
@@ -0,0 +1,23 @@
+class Solver
+  def reverse(string)
+    string.reverse
+  end
+
+  def fizzbuzz(number)
+    return 'fizzbuzz' if (number % 15).zero?
+    return 'fizz' if (number % 3).zero?
+    return 'buzz' if (number % 5).zero?
+
+    number
+  end
```

Comment:
- [ ] Based on the requirements, when a number is not divisible by `3` or `5` to give a remainder `0`, it should return the number as a **_string._** For example, the number `7` can't be divided by `3` or `5` and has a remainder of `0`, so it should `'7'` instead of `7`. Kindly correct this logic. 👍 💻

Reply:
Updated the `fizzbuzz` method